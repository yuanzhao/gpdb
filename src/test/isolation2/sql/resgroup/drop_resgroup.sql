1: SET ROLE TO r1;

0: ALTER ROLE r1 RESOURCE GROUP none;
0: DROP RESOURCE GROUP g1;

2: SET ROLE TO r2;

0: ALTER ROLE r2 RESOURCE GROUP none;
0: DROP RESOURCE GROUP g2;

! ls -d /sys/fs/cgroup/cpu/gpdb/*/;
! ls -d /sys/fs/cgroup/cpuacct/gpdb/*/;
