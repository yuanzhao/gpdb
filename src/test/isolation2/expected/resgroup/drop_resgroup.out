1: SET ROLE TO r1;
SET

0: ALTER ROLE r1 RESOURCE GROUP none;
ALTER
0: DROP RESOURCE GROUP g1;
DROP

2: SET ROLE TO r2;
SET

0: ALTER ROLE r2 RESOURCE GROUP none;
ALTER
0: DROP RESOURCE GROUP g2;
DROP

! ls -d /sys/fs/cgroup/cpu/gpdb/*/;
/sys/fs/cgroup/cpu/gpdb/6437/
/sys/fs/cgroup/cpu/gpdb/6438/

! ls -d /sys/fs/cgroup/cpuacct/gpdb/*/;
/sys/fs/cgroup/cpuacct/gpdb/6437/
/sys/fs/cgroup/cpuacct/gpdb/6438/

